app/src/docs/components/DocsLayout/Navigation/index.jsx

```diff
@@ -122,8 +145,14 @@ class Navigation extends React.Component<Props, State> {
                     container: responsive,
                 })}
                 >
-                    {!!responsive && this.parseCurrentPage()}
-                    {this.parseNavigation()}
+                    {responsive && (
+                        <li className={cx(styles.navListItem, styles.mobileHeader)}>
+                            <Link to="#">
+                                {this.generateMobileHeader()}
+                            </Link>
+                        </li>
+                    )}
+                    {Object.keys(docsNav).map((topLevelNav, index) => this.generateNavigation(topLevelNav, index))}
```

Comment:
if it were me, I'd make this a new component, rather than a method. Low priority.